source/_docs/settings-php.md

```diff
@@ -108,123 +108,32 @@ if (defined('PANTHEON_ENVIRONMENT')) {
-        $conf['preprocess_js'] = 0;
-      }
-
-      // Minimum cache lifetime - always none.
-      $conf['cache_lifetime'] = 0;
-      // Cached page compression - always off.
-      $conf['page_compression'] = 0;
-
-
-      if (PANTHEON_ENVIRONMENT == 'dev') {
-        // Google Analytics.
-        $conf['googleanalytics_account'] = 'UA-XXXXXXXX-X';
-      }
-      else if (PANTHEON_ENVIRONMENT == 'test') {
-        // Google Analytics.
-        $conf['googleanalytics_account'] = 'UA-XXXXXXXX-Y';
-      }
-      else if (PANTHEON_ENVIRONMENT == 'live') {
-        // Google Analytics.
-        $conf['googleanalytics_account'] = 'UA-XXXXXXXX-Z';
-      }
-    }
-
-The Drupal 8 [configuration override system](https://www.drupal.org/node/1928898) uses the global `$config` variable:
-
-    // All Pantheon Environments.
-    if (defined('PANTHEON_ENVIRONMENT')) {
-      // Drupal caching in test and live environments.
-      if (in_array(PANTHEON_ENVIRONMENT, array('test', 'live'))) {
-        // Expiration of cached pages - 15 minutes.
-        $config['system.performance']['cache']['page']['max_age'] = 900;
-        // Aggregate and compress CSS files in Drupal - on.
-        $config['system.performance']['css']['preprocess'] = true;
-        // Aggregate JavaScript files in Drupal - on.
-        $config['system.performance']['js']['preprocess'] = true;
-        // Google Analytics.
-        $config['google_analytics.settings']['account'] = 'UA-xxxxxxx-xx';
-      }
-      // Drupal caching on dev environment, and all multidevs
-      else {
-        // Expiration of cached pages - none.
-        $config['system.performance']['cache']['page']['max_age'] = 0;
-        // Aggregate and compress CSS files in Drupal - off.
-        $config['system.performance']['css']['preprocess'] = false;
-        // Aggregate JavaScript files in Drupal - off.
-        $config['system.performance']['js']['preprocess'] = false;
-      }
-    }
-
-#### How can I read the Pantheon environmental configuration, like database credentials?
-
-See [Reading the Pantheon Environment Configuration](/docs/read-environment-config).
-
-#### Why does Drupal report that `settings.php` is not protected? I can't change the permissions on `settings.php`.
-
-If you do not have a `settings.php` file in your codebase, you'll see the following message on `/admin/reports/status`:
-
+Yes, but only if at least one other file (e.g. `settings.php`) is present within the `sites/default` directory. Otherwise, the existing symlink to `sites/default/files` will be invalid.
+
+#### How can I write logic based on the Pantheon server environment?
+
+Depending on your use case, there are three possibilities:
+
+ - For web only actions, like redirects, check for the existence of `$_SERVER['PANTHEON_ENVIRONMENT']`. If it exists, it will contain a string with the current environment (Dev, Test, or Live. See our [redirects](/docs/redirects/#redirect-to-www) guide for examples.
```

Comment:
Note that `$_SERVER` is not generally available from the command line so [logic should check for that when used](https://pantheon.io/docs/redirects/#command-line-conditionals), and [avoid using `$_SERVER['SERVER_NAME']` and `$_SERVER['SERVER_PORT']`](https://pantheon.io/docs/server_name-and-server_port/)